aea/cli/install.py

```diff
@@ -92,7 +92,7 @@ def _install_dependency(dependency_name: str, dependency: Dependency):
         if return_code == 1:
             # try a second time
             return_code = _run_install_subprocess(command)
-        assert return_code == 0, "Return code != 0."
+        enforce(return_code == 0, "Return code != 0.")
```

Comment:
In cases where we try to enforce a condition other than for mypy it does fulfil its purpose.